### DISCUSSION TITLE
Dark Mode PDFs?

### DISCUSSION THREAD

**Discussion by ReaderGuy42** [chrisgrieser/shimmering-focus#289] [Q&A] [answered]
I can't find the option to toggle the inverting of the PDF color for dark mode in your theme. I thought I'd previously seen it. When I switch to Minimal theme it automatically inverts the PDF color, so I'm pretty sure none of my snippets are at fault here.

Thanks :)

**Comment by chrisgrieser** [comment]
That feature has been removed a while ago, since Obsidian introduced dark mode for PDFs on its own. Are you maybe on an old Obsidian version?

**Reply by ReaderGuy42** [reply]
No, I'm on 1.4.16. Where is this option supposed to be? Thanks!

**Reply by chrisgrieser** [reply, marked as answer]
Look here:
![image](https://github.com/chrisgrieser/shimmering-focus/assets/73286100/81e061bd-18a4-48b4-8ecf-7404e806098b)

**Reply by ReaderGuy42** [reply]
Perfect, thank you!!

**Comment by Drithyl** [comment]
Sorry for bringing this question back up - I can export dark mode PDFs with Obsidian's default theme, but trying to export a PDF with this theme's color/layout scheme doesn't seem to work. I know there's CSS that can be written specifically to modify the style of print, so I was wondering if that'll ever be done?

**Reply by chrisgrieser** [reply]
I just tried it, and when I export PDFs, the PDFs are always in light mode, regardless whether I use Shimmering Focus or the default theme. If the default theme exports in dark mode for you, you likely have some plugin installed that does that.

I think this is also intentional on Obsidian's side, since PDFs are often supposed to be printed. Also, most PDF readers have some feature that enable you to "invert" the colors of a PDF.

If you nonetheless want to have dark mode PDFs, I suggest you ask in the Obsidian Forum or Obsidian Discord, since I am not knowledgeable about that myself, and since this concerns Obsidian core as the default theme also exports PDFs in light mode.
